web/src/components/EditForm/EditForm.tsx

```diff
@@ -7,6 +9,33 @@ interface params {
 }
 
 const EditForm = ({ name, date, image }: params): JSX.Element => {
+  const [file, setFile] = useState<any>(null);
+  // const [photo, setPhoto] = useState<any>(null);
+
+  const sendFile = async (): Promise<any> => {
+    const formData = new FormData();
+    formData.append("doc", file);
+    // formData.append("photo", photo);
+
+    try {
+      await axios
+        .post("http://localhost:5050/upload/files", formData, {
+          headers: {
+            "Content-type": "multipart/form-data",
+          },
```

Comment:
Hardcoded URLS **should never** be injected in request. 

##### PRODUCTION SERVER WON'T KNOW WHERE LOCALHOST:5050 IS.

Reply:
Use `.env` variables. That's why they're needed and used.